### DISCUSSION TITLE
New strategy in naming conventions employed by blog sites?

### DISCUSSION THREAD

**Discussion by ms52538** [CodyBerenson/PGMA-Modernized#209] [Q&A]
I'm guessing you've already come across some interesting new naming conventions being used by blog sites.
Q*eerClik seems to be using a new technique which I and finding challenging to match, for example:

Say Uncle | Latin Leche: Jonas Matt, Felipe, Rodrigo El Santo – The Walls Are Thin – HAPPY HOLIDAYS SAVINGS!

First issue is the Studio Name: (Say Uncle _ Latin Leche); my software automatically converts the pipe symbol to become _  -and -
Second issue being the "HAPPY HOLIDAY SAVINGS!" trailing at the end (these are appearing randomly but more routinely of late on the site's blog posts).

Has anyone found a consistent way in which to deal with and overcome these (aside from manually editing the metadata directly, as in thru tag editing)?

**Comment by JPH71** [comment]
I will look into this as soon as possible as I will get a flood of issues...

Give me 3 or 4 titles so I can determine a pattern...

**Comment by ms52538** [comment]
It seems in part a promotional strategy by one blog site, unique studio naming conventions from multiple blog sites. I'm hoping it's a simple matter of stripping off the " - Promotional Blurb" found at the end of the name, and then a matter of deciphering how to format the studio names (almost like a "distributor name | studio name" approach).


[Say Uncle | Latin Leche: Jonas Matt and Felipe – At Your Service – SAVE ON MEMBERSHIP!](https://www.queerclick.com/qc/2022/10/say-uncle-latin-leche-jonas-matt-and-felipe-at-your-service.php)

[Next Door Studios: Scott Finn and Jayden Marcos flip fuck in “Hypothetically” (‘Next Door Films’ Series)](https://fagalicious.com/next-door-studios-scott-finn-jayden-marcos-flip-fuck-hypothetically-next-door-films-series/)

[Adult Time: Ty Roderick dominates and breeds Zak Bishop in “Mad Fuckers – Yes Sir, Thank You Sir” (Modern-Day Sins)](https://fagalicious.com/adult-time-ty-roderick-zak-bishop-mad-fuckers-yes-sir-thank-you-sir-modern-day-sins/)

[William Higgins: Jan Blatnik – SAVE UP TO 65%](https://www.queerclick.com/qc/2022/12/william-higgins-jan-blatnik-save-up-to-65.php)

[Say Uncle | Latin Leche: Pablo X, Rodrigo El Santo and Gonzo – A Stranger by the Lagoon Pt. 2](https://www.queerclick.com/qc/2022/10/say-uncle-latin-leche-pablo-x-rodrigo-el-santo-and-gonzo-a-stranger-by-the-lagoon-pt-2.php)

[NextDoorStudios: Dakota Payne and Jayden Marcos – Love Is Payne – Service Station – BLACK FRIDAY SAVINGS!](https://www.queerclick.com/qc/2022/11/nextdoorstudios-dakota-payne-and-jayden-marcos-love-is-payne-service-station-black-friday-savings.php)

[ChaosMen ‘Serviced’: VALENTIN PETROV Blows and Rims LARS LARSON ](https://www.waybig.com/blog/2022/12/14/chaosmen-serviced-valentin-petrov-blows-and-rims-lars-larson/)

[SayUncle: Dakota Lovell fucks Matthew Figata in “Hot for Gramps 2” (SayUncle Labs)](https://fagalicious.com/sayuncle-labs-dakota-lovell-fucks-matthew-figata-hot-for-gramps-2/)

**Reply by JPH71** [reply]
I have made changes to QueerClick to cover the Stuff ending like BLACK FRIDAY SAVINGS!, or SAVE UP TO 65%

For the pipe character **Say Uncle | Latin Leche** you should use either Say Uncle or Latin Leche. I named the scene like this
**_(Say Uncle) - Jonas Matt and Felipe – At Your Service (2022)_**
The code already caters for this.
Plex has space for only one Studio per film/scene - so choose either 

Next time - set this as an error rather than as a discussion as it prevented you matching the titles...

For the fagalicious scenes - those work and would match with no issues.....
You would need to explain what you want done with those and then I can try and code them in after speaking with Cody!

I will send the updated agents to Cody to post to the repository.....

**Comment by JPH71** [comment]
That is what I will do...
I will try and do it tomorrow

**Comment by firster520** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by ms52538** [comment]
Thank you very much, I appreciate the effort and time committed to this. It had not donned on me to post this in another area of the site, I'll try to follow the structure more closely in the future. 

Thanks again!
